screengrab/lib/screengrab/runner.rb

```diff
@@ -255,6 +255,7 @@ def run_tests_for_locale(locale, device_serial, test_classes_to_use, test_packag
       instrument_command = ["adb -s #{device_serial} shell am instrument --no-window-animation -w",
                             "-e testLocale #{locale.tr('-', '_')}",
                             "-e endingLocale #{@config[:ending_locale].tr('-', '_')}"]
+      instrument_command << "-e appendTimestamp #{@config[:use_timestamp_suffix] == true}"
```

Comment:
This `== true` can be removed with the addition of the `type: Boolean` mentioned above

Reply:
fixed 👍